controller/array_action/array_mediator_ds8k.py

```diff
@@ -415,10 +420,7 @@ def unmap_volume(self, volume_id, host_name):
             raise array_errors.UnMappingError(volume_id, host_name, ex.details)
 
     def get_flashcopies_by_volume(self, volume_id):
```

Comment:
do we really want to keep this method?

Reply:
yes, we need it..

Reply:
to save `.client` in the code?
why is it public anyway?

Reply:
because the Rest API have a bug and give all flashcopies in storage vol get volume, this give only the volume flashcopies

Reply:
IMO, as long as there is no further use for this function, there is no need to wrap the self.client.get_flashcopies_by_volume(volume_id) line in the get_flashcopies_by_volume function

Reply:
ok, tnx @annalisy!, I removed it